util/libcrypto.num

```diff
@@ -4298,3 +4298,4 @@ UI_dup_user_data                        4240	1_1_1	EXIST::FUNCTION:UI
 UI_method_get_data_destructor           4241	1_1_1	EXIST::FUNCTION:UI
 ERR_load_strings_const                  4242	1_1_1	EXIST::FUNCTION:
 ASN1_TIME_to_tm                         4243	1_1_1	EXIST::FUNCTION:
+ASN1_TIME_set_string_X509               4244 	1_1_1	EXIST::FUNCTION:
```

Comment:
From my understandings, it seems that 1.1.0 does not accept new features. Even if we backport it to 1.1.0, I don't know if the patch can merge into that branch without conflicts.